tests/phpunit/testcases/test_validation.php

```diff
@@ -63,4 +63,27 @@ function test_one_of() {
 		$this->assertEquals( true, $f( 3, array( 1, 2, 3 ) ) );
 		$this->assertEquals( false, $f( '1', array( 1, 2, 3 ) ) );
 	}
+
+	function test_is_ascii_string() {
+		$callback = GP_Validators::get( 'consisting_only_of_ASCII_characters' );
+		$f = $callback['positive'];
+		$this->assertEquals( true, $f( 'a' ) );
+		$this->assertEquals( true, $f( 'AbC' ) );
+		$this->assertEquals( true, $f( 'foo bar' ) );
+		$this->assertEquals( true, $f( '&' ) );
+		$this->assertEquals( true, $f( '123abc' ) );
+		$this->assertEquals( false, $f( 'äbc' ) );
+		$this->assertEquals( false, $f( 'ãbc' ) );
+	}
+
+	function test_is_starting_or_ending_with_a_word_character() {
```

Comment:
Rename to `test_is_starting_and_ending_with_a_word_character` to fix error message.